src/fitparser.cc

```diff
@@ -419,6 +437,13 @@ void FitParser::Encode(const v8::FunctionCallbackInfo<v8::Value> &args)
       lapMsg.SetTotalDistance(GET_LNUM("totalDistance"));
       lapMsg.SetMaxSpeed(GET_LNUM("maxSpeed"));
       lapMsg.SetAvgSpeed(GET_LNUM("avgSpeed"));
+      lapMsg.SetMaxCadence(GET_LNUM("maxCadence"));

```

Comment:
This is ok for now but as soon as we have multiple laps in a workout, we'll need to pass these in differently